### DISCUSSION TITLE
Outdated jQuery version

### DISCUSSION THREAD

**Discussion by sreejithitvlb** [locustio/locust#2720] [General]
Hello, I see Locust is using jQuery version 1.11.3 which has a DOS security violation. The latest version 3.6 doesn't have many other violations which I think good to use. Below link for reference...
https://security.snyk.io/package/npm/jquery
Do you think it's valid issue? and can I raise an issue for the same to upgrade the jQuery version ?

**Comment by cyberw** [comment]
Hi! Nice to see someone already found the discussion forum!

Normally I’d say go ahead an file an issue (no need to ask first :) But it has actually already been fixed in the new UI, and the old UI has even been removed completely now https://github.com/locustio/locust/pull/2703

**Comment by sreejithitvlb** [comment]
Thank you... You guys are awesome...
